






















































2019-03-29 11:01:23.875 27659 DEBUG oslo_messaging._drivers.amqpdriver [-] CALL msg_id: 79c9ffa470eb45c196e350fada0d4bd2 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:01:24.216 27659 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 79c9ffa470eb45c196e350fada0d4bd2 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:01:33.878 27659 DEBUG oslo_messaging._drivers.amqpdriver [-] CALL msg_id: c337c83a10cb4f73a676d354767fa397 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:01:33.909 27659 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: c337c83a10cb4f73a676d354767fa397 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:01:35.707 27659 DEBUG oslo_messaging._drivers.amqpdriver [-] received message msg_id: e7c42af54d214b5e8a9bba367bd31a57 reply to reply_85485a9a3d7c42638779a43e90afa2a8 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:255
2019-03-29 11:01:35.710 27659 DEBUG oslo_messaging._drivers.amqpdriver [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 30d421713d42462f99f7c7b305bf7d42 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:01:35.887 27659 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 30d421713d42462f99f7c7b305bf7d42 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:01:35.888 27659 DEBUG oslo_messaging._drivers.amqpdriver [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 6c3d2e448ebe4698955a3614fb1d44d2 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:01:35.898 27659 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 6c3d2e448ebe4698955a3614fb1d44d2 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:01:35.899 27659 DEBUG oslo_messaging._drivers.amqpdriver [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 89a19a40b824431a958cdf3d9e38c61c exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:01:35.908 27659 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 89a19a40b824431a958cdf3d9e38c61c __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:01:35.909 27659 DEBUG nova.virt.libvirt.driver [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Check if temp file /var/lib/nova/instances/tmpyRM7Eo exists to indicate shared storage is being used for migration. Exists? False _check_shared_storage_test_file /usr/lib/python2.7/site-packages/nova/virt/libvirt/driver.py:6057
2019-03-29 11:01:35.910 27659 DEBUG nova.compute.manager [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] source check data is LibvirtLiveMigrateData(bdms=<?>,block_migration=False,disk_available_mb=296960,disk_over_commit=False,filename='tmpyRM7Eo',graphics_listen_addr_spice=127.0.0.1,graphics_listen_addr_vnc=0.0.0.0,image_type='rbd',instance_relative_path='4ad84d2b-babc-4ef6-87eb-437ba51a77dc',is_shared_block_storage=True,is_shared_instance_path=False,is_volume_backed=False,migration=<?>,serial_listen_addr=None,serial_listen_ports=<?>,supported_perf_events=<?>,target_connect_addr=<?>) check_can_live_migrate_source /usr/lib/python2.7/site-packages/nova/compute/manager.py:5314
2019-03-29 11:01:35.911 27659 DEBUG oslo_messaging._drivers.amqpdriver [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: a173eb7e8373483880d0ce30b0e3b962 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:01:35.944 27659 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: a173eb7e8373483880d0ce30b0e3b962 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:01:35.944 27659 DEBUG oslo_messaging._drivers.amqpdriver [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] sending reply msg_id: e7c42af54d214b5e8a9bba367bd31a57 reply queue: reply_85485a9a3d7c42638779a43e90afa2a8 time elapsed: 0.236732568999s _send_reply /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:130
2019-03-29 11:01:35.999 27659 DEBUG oslo_messaging._drivers.amqpdriver [-] received message with unique_id: 3279d47f0e194fd1ad1f680993bed7eb __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:257
2019-03-29 11:01:36.002 27659 DEBUG oslo_messaging._drivers.amqpdriver [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 631339bc4d0147d398ab539370b28054 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:01:36.065 27659 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 631339bc4d0147d398ab539370b28054 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:01:36.066 27659 DEBUG oslo_messaging._drivers.amqpdriver [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 9dfeeee5ca674bb5be0b367c5b2d06ef exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:01:36.151 27659 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 9dfeeee5ca674bb5be0b367c5b2d06ef __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:01:36.152 27659 DEBUG oslo_messaging._drivers.amqpdriver [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 2e5e34864384458d96132a4c7f6dbde1 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:01:36.153 27659 DEBUG oslo_messaging._drivers.amqpdriver [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 2c90cbdf100841ed95b9ebf38a4d632d exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:01:36.202 27659 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 2e5e34864384458d96132a4c7f6dbde1 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:01:36.222 27659 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 2c90cbdf100841ed95b9ebf38a4d632d __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:01:36.223 27659 DEBUG oslo_messaging._drivers.amqpdriver [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 0753023940d942c28c5fafeaad0963ff exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:01:36.234 27659 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 0753023940d942c28c5fafeaad0963ff __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:01:36.270 27659 DEBUG oslo_messaging._drivers.amqpdriver [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 8d66fb7955384d2b87d3038f523c5e5e exchange 'nova' topic 'compute.com57' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:01:39.238 27659 DEBUG oslo_messaging._drivers.amqpdriver [-] received message with unique_id: d9fa8fe6ad0d438bbbdd5b8597c3e798 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:257
2019-03-29 11:01:39.241 27659 DEBUG nova.compute.manager [req-194cfe93-6eea-4f61-9ee6-f40b5f793eb2 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Received event network-vif-unplugged-7cf93e54-e1d8-4b03-a06b-7164797d1bcc external_instance_event /usr/lib/python2.7/site-packages/nova/compute/manager.py:6861
2019-03-29 11:01:39.241 27659 DEBUG oslo_concurrency.lockutils [req-194cfe93-6eea-4f61-9ee6-f40b5f793eb2 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] Lock "4ad84d2b-babc-4ef6-87eb-437ba51a77dc-events" acquired by "nova.compute.manager._pop_event" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2019-03-29 11:01:39.242 27659 DEBUG nova.compute.manager [req-194cfe93-6eea-4f61-9ee6-f40b5f793eb2 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Unexpected attempt to pop events during shutdown _pop_event /usr/lib/python2.7/site-packages/nova/compute/manager.py:350
2019-03-29 11:01:39.242 27659 DEBUG oslo_concurrency.lockutils [req-194cfe93-6eea-4f61-9ee6-f40b5f793eb2 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] Lock "4ad84d2b-babc-4ef6-87eb-437ba51a77dc-events" released by "nova.compute.manager._pop_event" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2019-03-29 11:01:39.243 27659 DEBUG nova.compute.manager [req-194cfe93-6eea-4f61-9ee6-f40b5f793eb2 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] No waiting events found dispatching network-vif-unplugged-7cf93e54-e1d8-4b03-a06b-7164797d1bcc pop_instance_event /usr/lib/python2.7/site-packages/nova/compute/manager.py:366
2019-03-29 11:01:39.243 27659 WARNING nova.compute.manager [req-194cfe93-6eea-4f61-9ee6-f40b5f793eb2 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Received unexpected event network-vif-unplugged-7cf93e54-e1d8-4b03-a06b-7164797d1bcc for instance
2019-03-29 11:01:39.678 27659 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 8d66fb7955384d2b87d3038f523c5e5e __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:01:39.681 27659 DEBUG oslo_messaging._drivers.amqpdriver [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 03d41ca1774640328a1573fb9cc99695 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:01:39.715 27659 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 03d41ca1774640328a1573fb9cc99695 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:01:39.716 27659 DEBUG nova.compute.manager [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] live_migration data is LibvirtLiveMigrateData(bdms=[],block_migration=False,disk_available_mb=296960,disk_over_commit=False,filename='tmpyRM7Eo',graphics_listen_addr_spice=127.0.0.1,graphics_listen_addr_vnc=0.0.0.0,image_type='rbd',instance_relative_path='4ad84d2b-babc-4ef6-87eb-437ba51a77dc',is_shared_block_storage=True,is_shared_instance_path=False,is_volume_backed=False,migration=Migration(57),serial_listen_addr=None,serial_listen_ports=[],supported_perf_events=[],target_connect_addr=None) _do_live_migration /usr/lib/python2.7/site-packages/nova/compute/manager.py:5422
2019-03-29 11:01:39.718 27659 DEBUG nova.virt.libvirt.driver [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Starting monitoring of live migration _live_migration /usr/lib/python2.7/site-packages/nova/virt/libvirt/driver.py:6609
2019-03-29 11:01:39.719 27659 DEBUG nova.objects.instance [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Lazy-loading 'flavor' on Instance uuid 4ad84d2b-babc-4ef6-87eb-437ba51a77dc obj_load_attr /usr/lib/python2.7/site-packages/nova/objects/instance.py:1034
2019-03-29 11:01:39.720 27659 DEBUG oslo_messaging._drivers.amqpdriver [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 0af0b22d8e894db285d9d0004b2c982f exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:01:39.733 27659 DEBUG oslo_messaging._drivers.amqpdriver [-] received message with unique_id: 36872049d8b54b3bb842cee8c9b6622a __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:257
2019-03-29 11:01:39.735 27659 DEBUG nova.compute.manager [req-9b42d0e2-8359-4b5c-9fa3-d2303e06c601 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Received event network-vif-plugged-7cf93e54-e1d8-4b03-a06b-7164797d1bcc external_instance_event /usr/lib/python2.7/site-packages/nova/compute/manager.py:6861
2019-03-29 11:01:39.736 27659 DEBUG oslo_concurrency.lockutils [req-9b42d0e2-8359-4b5c-9fa3-d2303e06c601 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] Lock "4ad84d2b-babc-4ef6-87eb-437ba51a77dc-events" acquired by "nova.compute.manager._pop_event" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2019-03-29 11:01:39.736 27659 DEBUG nova.compute.manager [req-9b42d0e2-8359-4b5c-9fa3-d2303e06c601 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Unexpected attempt to pop events during shutdown _pop_event /usr/lib/python2.7/site-packages/nova/compute/manager.py:350
2019-03-29 11:01:39.737 27659 DEBUG oslo_concurrency.lockutils [req-9b42d0e2-8359-4b5c-9fa3-d2303e06c601 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] Lock "4ad84d2b-babc-4ef6-87eb-437ba51a77dc-events" released by "nova.compute.manager._pop_event" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2019-03-29 11:01:39.737 27659 DEBUG nova.compute.manager [req-9b42d0e2-8359-4b5c-9fa3-d2303e06c601 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] No waiting events found dispatching network-vif-plugged-7cf93e54-e1d8-4b03-a06b-7164797d1bcc pop_instance_event /usr/lib/python2.7/site-packages/nova/compute/manager.py:366
2019-03-29 11:01:39.737 27659 WARNING nova.compute.manager [req-9b42d0e2-8359-4b5c-9fa3-d2303e06c601 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Received unexpected event network-vif-plugged-7cf93e54-e1d8-4b03-a06b-7164797d1bcc for instance
2019-03-29 11:01:39.773 27659 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 0af0b22d8e894db285d9d0004b2c982f __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:01:39.850 27659 DEBUG nova.virt.libvirt.migration [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Current None elapsed 0 steps [(0, 50), (150, 95), (300, 140), (450, 185), (600, 230), (750, 275), (900, 320), (1050, 365), (1200, 410), (1350, 455), (1500, 500)] update_downtime /usr/lib/python2.7/site-packages/nova/virt/libvirt/migration.py:346
2019-03-29 11:01:39.851 27659 INFO nova.virt.libvirt.migration [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Increasing downtime to 50 ms after 0 sec elapsed time
2019-03-29 11:01:39.854 27659 DEBUG oslo_messaging._drivers.amqpdriver [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: da0f755b27a64187a56e9ec776715ba8 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:01:39.888 27659 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: da0f755b27a64187a56e9ec776715ba8 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:01:39.891 27659 DEBUG oslo_messaging._drivers.amqpdriver [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 992fd4f330ea4c45a2aede6d410344df exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:01:40.004 27659 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 992fd4f330ea4c45a2aede6d410344df __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:01:40.006 27659 INFO nova.virt.libvirt.driver [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Migration running for 0 secs, memory 100% remaining; (bytes processed=0, remaining=0, total=0)
2019-03-29 11:01:40.313 27659 DEBUG nova.virt.driver [req-8a200d5c-47b1-48f4-a4ce-8f8af7bb7566 - - - - -] Emitting event <LifecycleEvent: 1553828500.31, 4ad84d2b-babc-4ef6-87eb-437ba51a77dc => Paused> emit_event /usr/lib/python2.7/site-packages/nova/virt/driver.py:1438
2019-03-29 11:01:40.314 27659 INFO nova.compute.manager [req-8a200d5c-47b1-48f4-a4ce-8f8af7bb7566 - - - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] VM Paused (Lifecycle Event)
2019-03-29 11:01:40.315 27659 DEBUG oslo_messaging._drivers.amqpdriver [req-8a200d5c-47b1-48f4-a4ce-8f8af7bb7566 - - - - -] CALL msg_id: 0401179153ae47e1a69d302842ba3141 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:01:40.354 27659 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 0401179153ae47e1a69d302842ba3141 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:01:40.357 27659 DEBUG nova.compute.manager [req-8a200d5c-47b1-48f4-a4ce-8f8af7bb7566 - - - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Checking state _get_power_state /usr/lib/python2.7/site-packages/nova/compute/manager.py:1176
2019-03-29 11:01:40.361 27659 DEBUG nova.compute.manager [req-8a200d5c-47b1-48f4-a4ce-8f8af7bb7566 - - - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Synchronizing instance power state after lifecycle event "Paused"; current vm_state: active, current task_state: migrating, current DB power_state: 1, VM power_state: 3 handle_lifecycle_event /usr/lib/python2.7/site-packages/nova/compute/manager.py:1088
2019-03-29 11:01:40.363 27659 DEBUG oslo_messaging._drivers.amqpdriver [req-8a200d5c-47b1-48f4-a4ce-8f8af7bb7566 - - - - -] CALL msg_id: 4511b2a7d345435ab50c87a4246d003c exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:01:40.426 27659 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 4511b2a7d345435ab50c87a4246d003c __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:01:40.427 27659 INFO nova.compute.manager [req-8a200d5c-47b1-48f4-a4ce-8f8af7bb7566 - - - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] During sync_power_state the instance has a pending task (migrating). Skip.
2019-03-29 11:01:40.588 27659 DEBUG nova.virt.libvirt.guest [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Domain has shutdown/gone away: Requested operation is not valid: domain is not running get_job_info /usr/lib/python2.7/site-packages/nova/virt/libvirt/guest.py:693
2019-03-29 11:01:40.589 27659 INFO nova.virt.libvirt.driver [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Migration operation has completed
2019-03-29 11:01:40.590 27659 INFO nova.compute.manager [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] _post_live_migration() is started..
2019-03-29 11:01:40.592 27659 DEBUG oslo_messaging._drivers.amqpdriver [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: b198302fa0bf4aafaea793f2f8879e3d exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:01:40.594 27659 DEBUG nova.virt.libvirt.driver [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Migration operation thread has finished _live_migration_operation /usr/lib/python2.7/site-packages/nova/virt/libvirt/driver.py:6295
2019-03-29 11:01:40.594 27659 DEBUG nova.virt.libvirt.driver [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Migration operation thread notification thread_finished /usr/lib/python2.7/site-packages/nova/virt/libvirt/driver.py:6600
2019-03-29 11:01:40.687 27659 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: b198302fa0bf4aafaea793f2f8879e3d __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:01:40.690 27659 DEBUG os_brick.utils [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] ==> get_connector_properties: call u"{'execute': None, 'my_ip': '172.30.126.58', 'enforce_multipath': True, 'host': 'com58', 'root_helper': 'sudo nova-rootwrap /etc/nova/rootwrap.conf', 'multipath': False}" trace_logging_wrapper /usr/lib/python2.7/site-packages/os_brick/utils.py:146
2019-03-29 11:01:40.692 27659 INFO oslo.privsep.daemon [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Running privsep helper: ['sudo', 'nova-rootwrap', '/etc/nova/rootwrap.conf', 'privsep-helper', '--config-file', '/usr/share/nova/nova-dist.conf', '--config-file', '/etc/nova/nova.conf', '--privsep_context', 'os_brick.privileged.default', '--privsep_sock_path', '/tmp/tmpEbEG70/privsep.sock']
2019-03-29 11:01:41.361 27659 INFO oslo.privsep.daemon [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Spawned new privsep daemon via rootwrap
2019-03-29 11:01:41.361 27659 DEBUG oslo.privsep.daemon [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Accepted privsep connection to /tmp/tmpEbEG70/privsep.sock __init__ /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:313
2019-03-29 11:01:41.363 27659 INFO oslo.privsep.daemon [-] privsep daemon starting
2019-03-29 11:01:41.363 27659 INFO oslo.privsep.daemon [-] privsep process running with uid/gid: 0/0
2019-03-29 11:01:41.363 27659 INFO oslo.privsep.daemon [-] privsep process running with capabilities (eff/prm/inh): CAP_SYS_ADMIN/CAP_SYS_ADMIN/none
2019-03-29 11:01:41.363 27659 INFO oslo.privsep.daemon [-] privsep daemon running as pid 28455
2019-03-29 11:01:41.364 27659 DEBUG oslo.privsep.daemon [-] privsep: request[77495984]: (1,) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 11:01:41.364 27659 DEBUG oslo.privsep.daemon [-] privsep: reply[77495984]: (2,) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 11:01:41.365 27659 DEBUG oslo.privsep.daemon [-] privsep: request[77495984]: (3, 'os_brick.privileged.rootwrap.execute_root', ('cat', '/etc/iscsi/initiatorname.iscsi'), {}) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 11:01:41.436 27659 DEBUG oslo.privsep.daemon [-] Running cmd (subprocess): cat /etc/iscsi/initiatorname.iscsi out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 11:01:41.439 27659 DEBUG oslo.privsep.daemon [-] CMD "cat /etc/iscsi/initiatorname.iscsi" returned: 0 in 0.003s out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 11:01:41.440 27659 DEBUG oslo.privsep.daemon [-] privsep: reply[77495984]: (4, ('InitiatorName=iqn.1994-05.com.redhat:9affbb458449\n', '')) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 11:01:41.440 27659 DEBUG os_brick.initiator.linuxfc [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] No Fibre Channel support detected on system. get_fc_hbas /usr/lib/python2.7/site-packages/os_brick/initiator/linuxfc.py:91
2019-03-29 11:01:41.441 27659 DEBUG os_brick.initiator.linuxfc [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] No Fibre Channel support detected on system. get_fc_hbas /usr/lib/python2.7/site-packages/os_brick/initiator/linuxfc.py:91
2019-03-29 11:01:41.441 27659 DEBUG os_brick.utils [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] <== get_connector_properties: return (749ms) {'initiator': u'iqn.1994-05.com.redhat:9affbb458449', 'ip': u'172.30.126.58', 'platform': u'x86_64', 'host': u'com58', 'do_local_attach': False, 'os_type': u'linux2', 'multipath': False} trace_logging_wrapper /usr/lib/python2.7/site-packages/os_brick/utils.py:170
2019-03-29 11:01:41.442 27659 DEBUG os_brick.utils [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] ==> get_connector_properties: call u"{'execute': None, 'my_ip': '172.30.126.58', 'enforce_multipath': True, 'host': 'com58', 'root_helper': 'sudo nova-rootwrap /etc/nova/rootwrap.conf', 'multipath': False}" trace_logging_wrapper /usr/lib/python2.7/site-packages/os_brick/utils.py:146
2019-03-29 11:01:41.442 27659 DEBUG oslo.privsep.daemon [-] privsep: request[77495984]: (3, 'os_brick.privileged.rootwrap.execute_root', ('cat', '/etc/iscsi/initiatorname.iscsi'), {}) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 11:01:41.443 27659 DEBUG oslo.privsep.daemon [-] Running cmd (subprocess): cat /etc/iscsi/initiatorname.iscsi out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 11:01:41.445 27659 DEBUG oslo.privsep.daemon [-] CMD "cat /etc/iscsi/initiatorname.iscsi" returned: 0 in 0.003s out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 11:01:41.446 27659 DEBUG oslo.privsep.daemon [-] privsep: reply[77495984]: (4, ('InitiatorName=iqn.1994-05.com.redhat:9affbb458449\n', '')) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 11:01:41.446 27659 DEBUG os_brick.initiator.linuxfc [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] No Fibre Channel support detected on system. get_fc_hbas /usr/lib/python2.7/site-packages/os_brick/initiator/linuxfc.py:91
2019-03-29 11:01:41.446 27659 DEBUG os_brick.initiator.linuxfc [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] No Fibre Channel support detected on system. get_fc_hbas /usr/lib/python2.7/site-packages/os_brick/initiator/linuxfc.py:91
2019-03-29 11:01:41.447 27659 DEBUG os_brick.utils [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] <== get_connector_properties: return (4ms) {'initiator': u'iqn.1994-05.com.redhat:9affbb458449', 'ip': u'172.30.126.58', 'platform': u'x86_64', 'host': u'com58', 'do_local_attach': False, 'os_type': u'linux2', 'multipath': False} trace_logging_wrapper /usr/lib/python2.7/site-packages/os_brick/utils.py:170
2019-03-29 11:01:41.447 27659 DEBUG oslo_concurrency.lockutils [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Acquired semaphore "refresh_cache-4ad84d2b-babc-4ef6-87eb-437ba51a77dc" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2019-03-29 11:01:41.448 27659 DEBUG nova.network.neutronv2.api [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] _get_instance_nw_info() _get_instance_nw_info /usr/lib/python2.7/site-packages/nova/network/neutronv2/api.py:1295
2019-03-29 11:01:41.449 27659 DEBUG oslo_messaging._drivers.amqpdriver [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 74b0479daf8c46408e5ed594ff8a706f exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:01:41.470 27659 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 74b0479daf8c46408e5ed594ff8a706f __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:01:41.481 27659 DEBUG oslo_messaging._drivers.amqpdriver [-] received message with unique_id: b816168673374363a84926a92a281444 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:257
2019-03-29 11:01:41.484 27659 DEBUG nova.compute.manager [req-102831f0-32aa-4032-99bd-c00f05903744 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Received event network-changed external_instance_event /usr/lib/python2.7/site-packages/nova/compute/manager.py:6861
2019-03-29 11:01:41.542 27659 DEBUG neutronclient.v2_0.client [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/ports.json?tenant_id=d4bbd972ca1b463ba028ccd1df983917&device_id=4ad84d2b-babc-4ef6-87eb-437ba51a77dc used request id req-4f1da4e9-fbbc-4537-beda-20b2b857dc68 _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 11:01:41.649 27659 DEBUG neutronclient.v2_0.client [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/networks.json?id=968d1118-8921-4ef9-9cf6-534ff3d8bf7d used request id req-a107c54a-ecb4-491e-a95d-2f3dc8f887dd _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 11:01:41.671 27659 DEBUG neutronclient.v2_0.client [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/floatingips.json?fixed_ip_address=10.10.10.3&port_id=7cf93e54-e1d8-4b03-a06b-7164797d1bcc used request id req-35d9dcfe-4a88-4c88-ac4e-8880999449c4 _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 11:01:41.733 27659 DEBUG neutronclient.v2_0.client [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/subnets.json?id=b9f5b815-a0ab-4476-a216-6b824fda2957 used request id req-2c8eed6d-e894-4e2b-9bb0-0c2cb12bc551 _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 11:01:41.778 27659 DEBUG neutronclient.v2_0.client [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/ports.json?network_id=968d1118-8921-4ef9-9cf6-534ff3d8bf7d&device_owner=network%3Adhcp used request id req-5007747a-74b2-4734-b979-087c84411920 _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 11:01:41.779 27659 DEBUG nova.network.base_api [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Updating instance_info_cache with network_info: [{"profile": {"migrating_to": "com57"}, "ovs_interfaceid": "7cf93e54-e1d8-4b03-a06b-7164797d1bcc", "preserve_on_delete": false, "network": {"bridge": "br-int", "subnets": [{"ips": [{"meta": {}, "version": 4, "type": "fixed", "floating_ips": [], "address": "10.10.10.3"}], "version": 4, "meta": {}, "dns": [], "routes": [], "cidr": "10.10.10.0/24", "gateway": {"meta": {}, "version": 4, "type": "gateway", "address": "10.10.10.1"}}], "meta": {"injected": false, "tenant_id": "d4bbd972ca1b463ba028ccd1df983917", "mtu": 1500}, "id": "968d1118-8921-4ef9-9cf6-534ff3d8bf7d", "label": "net10"}, "devname": "tap7cf93e54-e1", "vnic_type": "normal", "qbh_params": null, "meta": {}, "details": {"port_filter": true, "ovs_hybrid_plug": true}, "address": "fa:16:3e:73:f8:2c", "active": false, "type": "ovs", "id": "7cf93e54-e1d8-4b03-a06b-7164797d1bcc", "qbg_params": null}] update_instance_cache_with_nw_info /usr/lib/python2.7/site-packages/nova/network/base_api.py:48
2019-03-29 11:01:41.781 27659 DEBUG oslo_messaging._drivers.amqpdriver [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 66dca23dfb7649a98f3e77f84704d840 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:01:41.807 27659 DEBUG oslo_messaging._drivers.amqpdriver [-] received message with unique_id: 60628e5450ce4d10893c4d78749cea5d __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:257
2019-03-29 11:01:41.811 27659 DEBUG nova.compute.manager [req-3ea23c68-8c8e-45ff-a88b-e1f10491ac06 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Received event network-vif-plugged-7cf93e54-e1d8-4b03-a06b-7164797d1bcc external_instance_event /usr/lib/python2.7/site-packages/nova/compute/manager.py:6861
2019-03-29 11:01:41.812 27659 DEBUG oslo_concurrency.lockutils [req-3ea23c68-8c8e-45ff-a88b-e1f10491ac06 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] Lock "4ad84d2b-babc-4ef6-87eb-437ba51a77dc-events" acquired by "nova.compute.manager._pop_event" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2019-03-29 11:01:41.812 27659 DEBUG nova.compute.manager [req-3ea23c68-8c8e-45ff-a88b-e1f10491ac06 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Unexpected attempt to pop events during shutdown _pop_event /usr/lib/python2.7/site-packages/nova/compute/manager.py:350
2019-03-29 11:01:41.813 27659 DEBUG oslo_concurrency.lockutils [req-3ea23c68-8c8e-45ff-a88b-e1f10491ac06 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] Lock "4ad84d2b-babc-4ef6-87eb-437ba51a77dc-events" released by "nova.compute.manager._pop_event" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2019-03-29 11:01:41.813 27659 DEBUG nova.compute.manager [req-3ea23c68-8c8e-45ff-a88b-e1f10491ac06 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] No waiting events found dispatching network-vif-plugged-7cf93e54-e1d8-4b03-a06b-7164797d1bcc pop_instance_event /usr/lib/python2.7/site-packages/nova/compute/manager.py:366
2019-03-29 11:01:41.814 27659 WARNING nova.compute.manager [req-3ea23c68-8c8e-45ff-a88b-e1f10491ac06 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Received unexpected event network-vif-plugged-7cf93e54-e1d8-4b03-a06b-7164797d1bcc for instance
2019-03-29 11:01:41.819 27659 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 66dca23dfb7649a98f3e77f84704d840 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:01:41.820 27659 DEBUG oslo_concurrency.lockutils [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Releasing semaphore "refresh_cache-4ad84d2b-babc-4ef6-87eb-437ba51a77dc" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2019-03-29 11:01:41.820 27659 WARNING nova.image.glance [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] No protocol specified in for api_server 'http://172.30.126.56:9292', please update [glance] api_servers with fully qualified url including scheme (http / https)
2019-03-29 11:01:41.821 27659 DEBUG nova.compute.manager [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Calling driver.unfilter_instance from _post_live_migration _post_live_migration /usr/lib/python2.7/site-packages/nova/compute/manager.py:5605
2019-03-29 11:01:41.822 27659 DEBUG nova.virt.libvirt.vif [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] vif_type=ovs instance=Instance(access_ip_v4=None,access_ip_v6=None,architecture='x86_64',auto_disk_config=True,availability_zone='nova',cell_name=None,cleaned=False,config_drive='True',created_at=2019-03-29T02:58:47Z,default_ephemeral_device=None,default_swap_device=None,deleted=False,deleted_at=None,device_metadata=<?>,disable_terminate=False,display_description='vm003',display_name='vm003',ec2_ids=<?>,ephemeral_gb=0,ephemeral_key_uuid=None,fault=<?>,flavor=Flavor(1),host='com58',hostname='vm003',id=26,image_ref='40f27f13-d552-4f7e-ae5d-2b85b126c85e',info_cache=InstanceInfoCache,instance_type_id=1,kernel_id='',key_data=None,key_name=None,keypairs=<?>,launch_index=0,launched_at=2019-03-29T02:58:56Z,launched_on='com58',locked=False,locked_by=None,memory_mb=512,metadata={},migration_context=<?>,new_flavor=None,node='com58',numa_topology=<?>,old_flavor=None,os_type=None,pci_devices=<?>,pci_requests=<?>,power_state=1,progress=0,project_id='d4bbd972ca1b463ba028ccd1df983917',ramdisk_id='',reservation_id='r-aj6jz56r',root_device_name='/dev/vda',root_gb=1,security_groups=SecurityGroupList,services=<?>,shutdown_terminate=False,system_metadata={boot_roles='admin',image_architecture='x86_64',image_base_image_ref='40f27f13-d552-4f7e-ae5d-2b85b126c85e',image_container_format='bare',image_disk_format='qcow2',image_min_disk='1',image_min_ram='0'},tags=<?>,task_state='migrating',terminated_at=None,updated_at=2019-03-29T03:01:35Z,user_data=None,user_id='9e9e53917dda421083fa6af425c6bfbf',uuid=4ad84d2b-babc-4ef6-87eb-437ba51a77dc,vcpu_model=<?>,vcpus=1,vm_mode=None,vm_state='active') vif={"profile": {"migrating_to": "com57"}, "ovs_interfaceid": "7cf93e54-e1d8-4b03-a06b-7164797d1bcc", "preserve_on_delete": false, "network": {"bridge": "br-int", "subnets": [{"ips": [{"version": 4, "vif_mac": "fa:16:3e:73:f8:2c", "floating_ips": [], "label": "net10", "meta": {}, "address": "10.10.10.3", "type": "fixed"}], "version": 4, "meta": {}, "dns": [], "routes": [], "cidr": "10.10.10.0/24", "gateway": {"meta": {}, "version": 4, "type": "gateway", "address": "10.10.10.1"}}], "meta": {"injected": false, "tenant_id": "d4bbd972ca1b463ba028ccd1df983917", "mtu": 1500}, "id": "968d1118-8921-4ef9-9cf6-534ff3d8bf7d", "label": "net10"}, "devname": "tap7cf93e54-e1", "vnic_type": "normal", "qbh_params": null, "meta": {}, "details": {"port_filter": true, "ovs_hybrid_plug": true}, "address": "fa:16:3e:73:f8:2c", "active": false, "type": "ovs", "id": "7cf93e54-e1d8-4b03-a06b-7164797d1bcc", "qbg_params": null} unplug /usr/lib/python2.7/site-packages/nova/virt/libvirt/vif.py:966
2019-03-29 11:01:41.823 27659 DEBUG nova.network.os_vif_util [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Converting VIF {"profile": {"migrating_to": "com57"}, "ovs_interfaceid": "7cf93e54-e1d8-4b03-a06b-7164797d1bcc", "preserve_on_delete": false, "network": {"bridge": "br-int", "subnets": [{"ips": [{"version": 4, "vif_mac": "fa:16:3e:73:f8:2c", "floating_ips": [], "label": "net10", "meta": {}, "address": "10.10.10.3", "type": "fixed"}], "version": 4, "meta": {}, "dns": [], "routes": [], "cidr": "10.10.10.0/24", "gateway": {"meta": {}, "version": 4, "type": "gateway", "address": "10.10.10.1"}}], "meta": {"injected": false, "tenant_id": "d4bbd972ca1b463ba028ccd1df983917", "mtu": 1500}, "id": "968d1118-8921-4ef9-9cf6-534ff3d8bf7d", "label": "net10"}, "devname": "tap7cf93e54-e1", "vnic_type": "normal", "qbh_params": null, "meta": {}, "details": {"port_filter": true, "ovs_hybrid_plug": true}, "address": "fa:16:3e:73:f8:2c", "active": false, "type": "ovs", "id": "7cf93e54-e1d8-4b03-a06b-7164797d1bcc", "qbg_params": null} nova_to_osvif_vif /usr/lib/python2.7/site-packages/nova/network/os_vif_util.py:413
2019-03-29 11:01:41.823 27659 DEBUG nova.network.os_vif_util [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Converted object VIFBridge(active=False,address=fa:16:3e:73:f8:2c,bridge_name='qbr7cf93e54-e1',has_traffic_filtering=True,id=7cf93e54-e1d8-4b03-a06b-7164797d1bcc,network=Network(968d1118-8921-4ef9-9cf6-534ff3d8bf7d),plugin='ovs',port_profile=VIFPortProfileBase,preserve_on_delete=False,vif_name='tap7cf93e54-e1') nova_to_osvif_vif /usr/lib/python2.7/site-packages/nova/network/os_vif_util.py:425
2019-03-29 11:01:41.824 27659 DEBUG os_vif [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Unplugging vif VIFBridge(active=False,address=fa:16:3e:73:f8:2c,bridge_name='qbr7cf93e54-e1',has_traffic_filtering=True,id=7cf93e54-e1d8-4b03-a06b-7164797d1bcc,network=Network(968d1118-8921-4ef9-9cf6-534ff3d8bf7d),plugin='ovs',port_profile=VIFPortProfileBase,preserve_on_delete=False,vif_name='tap7cf93e54-e1') unplug /usr/lib/python2.7/site-packages/os_vif/__init__.py:112
2019-03-29 11:01:41.825 27659 DEBUG oslo_concurrency.lockutils [req-102831f0-32aa-4032-99bd-c00f05903744 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] Acquired semaphore "refresh_cache-4ad84d2b-babc-4ef6-87eb-437ba51a77dc" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2019-03-29 11:01:41.827 27659 DEBUG nova.network.neutronv2.api [req-102831f0-32aa-4032-99bd-c00f05903744 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] _get_instance_nw_info() _get_instance_nw_info /usr/lib/python2.7/site-packages/nova/network/neutronv2/api.py:1295
2019-03-29 11:01:41.828 27659 DEBUG oslo_messaging._drivers.amqpdriver [req-102831f0-32aa-4032-99bd-c00f05903744 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] CALL msg_id: baa9f7fcd7b048d69e9930da00ce6ca0 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:01:41.829 27659 DEBUG oslo.privsep.daemon [-] privsep: request[77495984]: (3, 'vif_plug_ovs.linux_net.delete_bridge', (u'qbr7cf93e54-e1', 'qvb7cf93e54-e1'), {}) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 11:01:41.829 27659 DEBUG oslo.privsep.daemon [-] Running cmd (subprocess): brctl delif qbr7cf93e54-e1 qvb7cf93e54-e1 out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 11:01:41.837 27659 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: baa9f7fcd7b048d69e9930da00ce6ca0 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:01:41.844 27659 DEBUG oslo.privsep.daemon [-] CMD "brctl delif qbr7cf93e54-e1 qvb7cf93e54-e1" returned: 0 in 0.017s out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 11:01:41.844 27659 DEBUG oslo.privsep.daemon [-] Running cmd (subprocess): ip link set qbr7cf93e54-e1 down out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 11:01:41.851 27659 DEBUG oslo.privsep.daemon [-] CMD "ip link set qbr7cf93e54-e1 down" returned: 0 in 0.007s out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 11:01:41.851 27659 DEBUG oslo.privsep.daemon [-] Running cmd (subprocess): brctl delbr qbr7cf93e54-e1 out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 11:01:41.865 27659 DEBUG oslo.privsep.daemon [-] CMD "brctl delbr qbr7cf93e54-e1" returned: 0 in 0.014s out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 11:01:41.866 27659 DEBUG oslo.privsep.daemon [-] privsep: reply[77495984]: (4, None) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 11:01:41.868 27659 DEBUG oslo.privsep.daemon [-] privsep: request[77495984]: (3, 'vif_plug_ovs.linux_net.delete_ovs_vif_port', (u'br-int', 'qvo7cf93e54-e1'), {'timeout': 120}) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 11:01:41.868 27659 DEBUG oslo.privsep.daemon [-] Running cmd (subprocess): ovs-vsctl --timeout=120 -- --if-exists del-port br-int qvo7cf93e54-e1 out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 11:01:41.891 27659 DEBUG oslo.privsep.daemon [-] CMD "ovs-vsctl --timeout=120 -- --if-exists del-port br-int qvo7cf93e54-e1" returned: 0 in 0.022s out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 11:01:41.891 27659 DEBUG oslo.privsep.daemon [-] Running cmd (subprocess): ip link delete qvo7cf93e54-e1 out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 11:01:41.899 27659 DEBUG neutronclient.v2_0.client [req-102831f0-32aa-4032-99bd-c00f05903744 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/ports.json?tenant_id=d4bbd972ca1b463ba028ccd1df983917&device_id=4ad84d2b-babc-4ef6-87eb-437ba51a77dc used request id req-2fa4a2ae-0984-43f1-849d-a40f9db38bb1 _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 11:01:41.907 27659 DEBUG oslo.privsep.daemon [-] CMD "ip link delete qvo7cf93e54-e1" returned: 0 in 0.016s out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 11:01:41.907 27659 DEBUG oslo.privsep.daemon [-] Net device removed: 'qvo7cf93e54-e1' out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 11:01:41.907 27659 DEBUG oslo.privsep.daemon [-] privsep: reply[77495984]: (4, None) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 11:01:41.908 27659 INFO os_vif [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Successfully unplugged vif VIFBridge(active=False,address=fa:16:3e:73:f8:2c,bridge_name='qbr7cf93e54-e1',has_traffic_filtering=True,id=7cf93e54-e1d8-4b03-a06b-7164797d1bcc,network=Network(968d1118-8921-4ef9-9cf6-534ff3d8bf7d),plugin='ovs',port_profile=VIFPortProfileBase,preserve_on_delete=False,vif_name='tap7cf93e54-e1')
2019-03-29 11:01:41.909 27659 DEBUG oslo_messaging._drivers.amqpdriver [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: e4882dce7a8345638dfbea68f068a631 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:01:41.926 27659 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: e4882dce7a8345638dfbea68f068a631 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:01:41.931 27659 DEBUG oslo_messaging._drivers.amqpdriver [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: a8ad7e11757646c581e801d0011ebde1 exchange 'nova' topic 'compute.com57' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:01:42.177 27659 DEBUG neutronclient.v2_0.client [req-102831f0-32aa-4032-99bd-c00f05903744 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/networks.json?id=968d1118-8921-4ef9-9cf6-534ff3d8bf7d used request id req-44d5eef0-f67e-47c7-9787-782f9b75d01a _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 11:01:42.199 27659 DEBUG neutronclient.v2_0.client [req-102831f0-32aa-4032-99bd-c00f05903744 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/floatingips.json?fixed_ip_address=10.10.10.3&port_id=7cf93e54-e1d8-4b03-a06b-7164797d1bcc used request id req-d5c5dc79-8f2e-4c01-b4b8-515967747763 _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 11:01:42.319 27659 DEBUG neutronclient.v2_0.client [req-102831f0-32aa-4032-99bd-c00f05903744 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/subnets.json?id=b9f5b815-a0ab-4476-a216-6b824fda2957 used request id req-c0a11451-07d9-46c9-89e6-6943fad6d074 _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 11:01:42.386 27659 DEBUG neutronclient.v2_0.client [req-102831f0-32aa-4032-99bd-c00f05903744 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/ports.json?network_id=968d1118-8921-4ef9-9cf6-534ff3d8bf7d&device_owner=network%3Adhcp used request id req-2dceccfc-a68f-4bb2-a4e0-a46945a59008 _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 11:01:42.387 27659 DEBUG nova.network.base_api [req-102831f0-32aa-4032-99bd-c00f05903744 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Updating instance_info_cache with network_info: [{"profile": {"migrating_to": "com57"}, "ovs_interfaceid": "7cf93e54-e1d8-4b03-a06b-7164797d1bcc", "preserve_on_delete": false, "network": {"bridge": "br-int", "subnets": [{"ips": [{"meta": {}, "version": 4, "type": "fixed", "floating_ips": [], "address": "10.10.10.3"}], "version": 4, "meta": {}, "dns": [], "routes": [], "cidr": "10.10.10.0/24", "gateway": {"meta": {}, "version": 4, "type": "gateway", "address": "10.10.10.1"}}], "meta": {"injected": false, "tenant_id": "d4bbd972ca1b463ba028ccd1df983917", "mtu": 1500}, "id": "968d1118-8921-4ef9-9cf6-534ff3d8bf7d", "label": "net10"}, "devname": "tap7cf93e54-e1", "vnic_type": "normal", "qbh_params": null, "meta": {}, "details": {"port_filter": true, "ovs_hybrid_plug": true}, "address": "fa:16:3e:73:f8:2c", "active": true, "type": "ovs", "id": "7cf93e54-e1d8-4b03-a06b-7164797d1bcc", "qbg_params": null}] update_instance_cache_with_nw_info /usr/lib/python2.7/site-packages/nova/network/base_api.py:48
2019-03-29 11:01:42.388 27659 DEBUG oslo_messaging._drivers.amqpdriver [req-102831f0-32aa-4032-99bd-c00f05903744 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] CALL msg_id: ec95ce6ceab745d39d3308ed6ea0ed65 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:01:42.444 27659 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: ec95ce6ceab745d39d3308ed6ea0ed65 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:01:42.445 27659 DEBUG oslo_concurrency.lockutils [req-102831f0-32aa-4032-99bd-c00f05903744 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] Releasing semaphore "refresh_cache-4ad84d2b-babc-4ef6-87eb-437ba51a77dc" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2019-03-29 11:01:43.577 27659 DEBUG oslo_messaging._drivers.amqpdriver [-] received message with unique_id: adebba7c13904a13b424cd9f5877a050 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:257
2019-03-29 11:01:43.579 27659 DEBUG nova.compute.manager [req-b5a16225-053b-48de-95b1-c5b1c21638f8 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Received event network-changed external_instance_event /usr/lib/python2.7/site-packages/nova/compute/manager.py:6861
2019-03-29 11:01:43.580 27659 DEBUG oslo_concurrency.lockutils [req-b5a16225-053b-48de-95b1-c5b1c21638f8 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] Acquired semaphore "refresh_cache-4ad84d2b-babc-4ef6-87eb-437ba51a77dc" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2019-03-29 11:01:43.580 27659 DEBUG nova.network.neutronv2.api [req-b5a16225-053b-48de-95b1-c5b1c21638f8 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] _get_instance_nw_info() _get_instance_nw_info /usr/lib/python2.7/site-packages/nova/network/neutronv2/api.py:1295
2019-03-29 11:01:43.581 27659 DEBUG oslo_messaging._drivers.amqpdriver [req-b5a16225-053b-48de-95b1-c5b1c21638f8 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] CALL msg_id: c6ea377224b64a19a761b5daa49b09d9 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:01:43.591 27659 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: c6ea377224b64a19a761b5daa49b09d9 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:01:43.751 27659 DEBUG neutronclient.v2_0.client [req-b5a16225-053b-48de-95b1-c5b1c21638f8 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/ports.json?tenant_id=d4bbd972ca1b463ba028ccd1df983917&device_id=4ad84d2b-babc-4ef6-87eb-437ba51a77dc used request id req-76914744-9427-4798-abca-11f3b63ff7fe _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 11:01:43.881 27659 DEBUG oslo_messaging._drivers.amqpdriver [-] CALL msg_id: 8609d6f02ef34fe8b2fb4f36a9a618d6 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:01:43.884 27659 DEBUG oslo_messaging._drivers.amqpdriver [-] received message with unique_id: 3ed8fb1d169b47a4a94d59d6991e4693 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:257
2019-03-29 11:01:43.887 27659 DEBUG nova.compute.manager [req-0a0b9ac8-5b56-4add-b0c6-81f887700037 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Received event network-vif-plugged-7cf93e54-e1d8-4b03-a06b-7164797d1bcc external_instance_event /usr/lib/python2.7/site-packages/nova/compute/manager.py:6861
2019-03-29 11:01:43.887 27659 DEBUG oslo_concurrency.lockutils [req-0a0b9ac8-5b56-4add-b0c6-81f887700037 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] Lock "4ad84d2b-babc-4ef6-87eb-437ba51a77dc-events" acquired by "nova.compute.manager._pop_event" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2019-03-29 11:01:43.888 27659 DEBUG nova.compute.manager [req-0a0b9ac8-5b56-4add-b0c6-81f887700037 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Unexpected attempt to pop events during shutdown _pop_event /usr/lib/python2.7/site-packages/nova/compute/manager.py:350
2019-03-29 11:01:43.888 27659 DEBUG oslo_concurrency.lockutils [req-0a0b9ac8-5b56-4add-b0c6-81f887700037 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] Lock "4ad84d2b-babc-4ef6-87eb-437ba51a77dc-events" released by "nova.compute.manager._pop_event" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2019-03-29 11:01:43.888 27659 DEBUG nova.compute.manager [req-0a0b9ac8-5b56-4add-b0c6-81f887700037 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] No waiting events found dispatching network-vif-plugged-7cf93e54-e1d8-4b03-a06b-7164797d1bcc pop_instance_event /usr/lib/python2.7/site-packages/nova/compute/manager.py:366
2019-03-29 11:01:43.889 27659 WARNING nova.compute.manager [req-0a0b9ac8-5b56-4add-b0c6-81f887700037 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Received unexpected event network-vif-plugged-7cf93e54-e1d8-4b03-a06b-7164797d1bcc for instance
2019-03-29 11:01:43.940 27659 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 8609d6f02ef34fe8b2fb4f36a9a618d6 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:01:43.943 27659 DEBUG neutronclient.v2_0.client [req-b5a16225-053b-48de-95b1-c5b1c21638f8 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/networks.json?id=968d1118-8921-4ef9-9cf6-534ff3d8bf7d used request id req-c9d9560e-74ce-4f12-a643-980c24e11eba _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 11:01:43.983 27659 DEBUG neutronclient.v2_0.client [req-b5a16225-053b-48de-95b1-c5b1c21638f8 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/floatingips.json?fixed_ip_address=10.10.10.3&port_id=7cf93e54-e1d8-4b03-a06b-7164797d1bcc used request id req-71366f7e-2b46-48ec-9f52-440dab52cf4d _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 11:01:44.470 27659 DEBUG neutronclient.v2_0.client [req-b5a16225-053b-48de-95b1-c5b1c21638f8 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/subnets.json?id=b9f5b815-a0ab-4476-a216-6b824fda2957 used request id req-cb999a6d-c078-4806-94eb-cd197de6f098 _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 11:01:44.524 27659 DEBUG neutronclient.v2_0.client [req-b5a16225-053b-48de-95b1-c5b1c21638f8 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/ports.json?network_id=968d1118-8921-4ef9-9cf6-534ff3d8bf7d&device_owner=network%3Adhcp used request id req-9fdeba7d-dd2a-40ff-aaf1-3d8de797083b _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 11:01:44.525 27659 DEBUG nova.network.base_api [req-b5a16225-053b-48de-95b1-c5b1c21638f8 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Updating instance_info_cache with network_info: [{"profile": {}, "ovs_interfaceid": null, "preserve_on_delete": false, "network": {"bridge": null, "subnets": [{"ips": [{"meta": {}, "version": 4, "type": "fixed", "floating_ips": [], "address": "10.10.10.3"}], "version": 4, "meta": {}, "dns": [], "routes": [], "cidr": "10.10.10.0/24", "gateway": {"meta": {}, "version": 4, "type": "gateway", "address": "10.10.10.1"}}], "meta": {"injected": false, "tenant_id": "d4bbd972ca1b463ba028ccd1df983917", "mtu": 1500}, "id": "968d1118-8921-4ef9-9cf6-534ff3d8bf7d", "label": "net10"}, "devname": "tap7cf93e54-e1", "vnic_type": "normal", "qbh_params": null, "meta": {}, "details": {}, "address": "fa:16:3e:73:f8:2c", "active": false, "type": "unbound", "id": "7cf93e54-e1d8-4b03-a06b-7164797d1bcc", "qbg_params": null}] update_instance_cache_with_nw_info /usr/lib/python2.7/site-packages/nova/network/base_api.py:48
2019-03-29 11:01:44.526 27659 DEBUG oslo_messaging._drivers.amqpdriver [req-b5a16225-053b-48de-95b1-c5b1c21638f8 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] CALL msg_id: 293fda3b17f9488386c82bcb8c3d3888 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:01:44.590 27659 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 293fda3b17f9488386c82bcb8c3d3888 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:01:44.590 27659 DEBUG oslo_concurrency.lockutils [req-b5a16225-053b-48de-95b1-c5b1c21638f8 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] Releasing semaphore "refresh_cache-4ad84d2b-babc-4ef6-87eb-437ba51a77dc" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2019-03-29 11:01:44.591 27659 DEBUG nova.compute.manager [req-b5a16225-053b-48de-95b1-c5b1c21638f8 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Received event network-vif-unplugged-7cf93e54-e1d8-4b03-a06b-7164797d1bcc external_instance_event /usr/lib/python2.7/site-packages/nova/compute/manager.py:6861
2019-03-29 11:01:44.591 27659 DEBUG oslo_concurrency.lockutils [req-b5a16225-053b-48de-95b1-c5b1c21638f8 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] Lock "4ad84d2b-babc-4ef6-87eb-437ba51a77dc-events" acquired by "nova.compute.manager._pop_event" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2019-03-29 11:01:44.592 27659 DEBUG nova.compute.manager [req-b5a16225-053b-48de-95b1-c5b1c21638f8 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Unexpected attempt to pop events during shutdown _pop_event /usr/lib/python2.7/site-packages/nova/compute/manager.py:350
2019-03-29 11:01:44.592 27659 DEBUG oslo_concurrency.lockutils [req-b5a16225-053b-48de-95b1-c5b1c21638f8 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] Lock "4ad84d2b-babc-4ef6-87eb-437ba51a77dc-events" released by "nova.compute.manager._pop_event" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2019-03-29 11:01:44.593 27659 DEBUG nova.compute.manager [req-b5a16225-053b-48de-95b1-c5b1c21638f8 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] No waiting events found dispatching network-vif-unplugged-7cf93e54-e1d8-4b03-a06b-7164797d1bcc pop_instance_event /usr/lib/python2.7/site-packages/nova/compute/manager.py:366
2019-03-29 11:01:44.593 27659 WARNING nova.compute.manager [req-b5a16225-053b-48de-95b1-c5b1c21638f8 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Received unexpected event network-vif-unplugged-7cf93e54-e1d8-4b03-a06b-7164797d1bcc for instance
2019-03-29 11:01:45.206 27659 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: a8ad7e11757646c581e801d0011ebde1 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:01:45.207 27659 DEBUG nova.compute.manager [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Calling driver.cleanup from _post_live_migration _post_live_migration /usr/lib/python2.7/site-packages/nova/compute/manager.py:5645
2019-03-29 11:01:45.209 27659 DEBUG oslo_concurrency.processutils [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Running cmd (subprocess): mv /var/lib/nova/instances/4ad84d2b-babc-4ef6-87eb-437ba51a77dc /var/lib/nova/instances/4ad84d2b-babc-4ef6-87eb-437ba51a77dc_del execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:355
2019-03-29 11:01:45.227 27659 DEBUG oslo_concurrency.processutils [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CMD "mv /var/lib/nova/instances/4ad84d2b-babc-4ef6-87eb-437ba51a77dc /var/lib/nova/instances/4ad84d2b-babc-4ef6-87eb-437ba51a77dc_del" returned: 0 in 0.019s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:385
2019-03-29 11:01:45.230 27659 INFO nova.virt.libvirt.driver [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Deleting instance files /var/lib/nova/instances/4ad84d2b-babc-4ef6-87eb-437ba51a77dc_del
2019-03-29 11:01:45.231 27659 INFO nova.virt.libvirt.driver [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Deletion of /var/lib/nova/instances/4ad84d2b-babc-4ef6-87eb-437ba51a77dc_del complete
2019-03-29 11:01:45.235 27659 DEBUG oslo_messaging._drivers.amqpdriver [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 290a1c5d250d470189263e4a7ce0d714 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:01:45.318 27659 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 290a1c5d250d470189263e4a7ce0d714 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:01:45.323 27659 DEBUG oslo_concurrency.lockutils [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Lock "4ad84d2b-babc-4ef6-87eb-437ba51a77dc-events" acquired by "nova.compute.manager._clear_events" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2019-03-29 11:01:45.324 27659 DEBUG oslo_concurrency.lockutils [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Lock "4ad84d2b-babc-4ef6-87eb-437ba51a77dc-events" released by "nova.compute.manager._clear_events" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2019-03-29 11:01:45.325 27659 DEBUG oslo_messaging._drivers.amqpdriver [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 93f1fa8c4fbe46169b03112022c9edac exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:01:45.382 27659 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 93f1fa8c4fbe46169b03112022c9edac __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:01:45.385 27659 DEBUG nova.compute.resource_tracker [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Auditing locally available compute resources for com58 (node: com58) update_available_resource /usr/lib/python2.7/site-packages/nova/compute/resource_tracker.py:534
2019-03-29 11:01:45.503 27659 DEBUG nova.compute.resource_tracker [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Hypervisor: free VCPUs: 2 _report_hypervisor_resource_view /usr/lib/python2.7/site-packages/nova/compute/resource_tracker.py:655
2019-03-29 11:01:45.504 27659 DEBUG nova.compute.resource_tracker [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Hypervisor/Node resource view: name=com58 free_ram=7108MB free_disk=290GB free_vcpus=2 pci_devices=[{"dev_id": "pci_0000_00_00_0", "product_id": "7190", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "8086", "label": "label_8086_7190", "address": "0000:00:00.0"}, {"dev_id": "pci_0000_00_01_0", "product_id": "7191", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "8086", "label": "label_8086_7191", "address": "0000:00:01.0"}, {"dev_id": "pci_0000_00_07_0", "product_id": "7110", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "8086", "label": "label_8086_7110", "address": "0000:00:07.0"}, {"dev_id": "pci_0000_00_07_1", "product_id": "7111", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "8086", "label": "label_8086_7111", "address": "0000:00:07.1"}, {"dev_id": "pci_0000_00_07_3", "product_id": "7113", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "8086", "label": "label_8086_7113", "address": "0000:00:07.3"}, {"dev_id": "pci_0000_00_07_7", "product_id": "0740", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_0740", "address": "0000:00:07.7"}, {"dev_id": "pci_0000_00_0f_0", "product_id": "0405", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_0405", "address": "0000:00:0f.0"}, {"dev_id": "pci_0000_00_11_0", "product_id": "0790", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_0790", "address": "0000:00:11.0"}, {"dev_id": "pci_0000_00_15_0", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:15.0"}, {"dev_id": "pci_0000_03_00_0", "product_id": "07c0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07c0", "address": "0000:03:00.0"}, {"dev_id": "pci_0000_00_15_1", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:15.1"}, {"dev_id": "pci_0000_00_15_2", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:15.2"}, {"dev_id": "pci_0000_00_15_3", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:15.3"}, {"dev_id": "pci_0000_00_15_4", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:15.4"}, {"dev_id": "pci_0000_00_15_5", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:15.5"}, {"dev_id": "pci_0000_00_15_6", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:15.6"}, {"dev_id": "pci_0000_00_15_7", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:15.7"}, {"dev_id": "pci_0000_00_16_0", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:16.0"}, {"dev_id": "pci_0000_0b_00_0", "product_id": "07b0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07b0", "address": "0000:0b:00.0"}, {"dev_id": "pci_0000_00_16_1", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:16.1"}, {"dev_id": "pci_0000_00_16_2", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:16.2"}, {"dev_id": "pci_0000_00_16_3", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:16.3"}, {"dev_id": "pci_0000_00_16_4", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:16.4"}, {"dev_id": "pci_0000_00_16_5", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:16.5"}, {"dev_id": "pci_0000_00_16_6", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:16.6"}, {"dev_id": "pci_0000_00_16_7", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:16.7"}, {"dev_id": "pci_0000_00_17_0", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:17.0"}, {"dev_id": "pci_0000_13_00_0", "product_id": "07b0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07b0", "address": "0000:13:00.0"}, {"dev_id": "pci_0000_00_17_1", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:17.1"}, {"dev_id": "pci_0000_00_17_2", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:17.2"}, {"dev_id": "pci_0000_00_17_3", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:17.3"}, {"dev_id": "pci_0000_00_17_4", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:17.4"}, {"dev_id": "pci_0000_00_17_5", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:17.5"}, {"dev_id": "pci_0000_00_17_6", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:17.6"}, {"dev_id": "pci_0000_00_17_7", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:17.7"}, {"dev_id": "pci_0000_00_18_0", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:18.0"}, {"dev_id": "pci_0000_00_18_1", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:18.1"}, {"dev_id": "pci_0000_00_18_2", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:18.2"}, {"dev_id": "pci_0000_00_18_3", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:18.3"}, {"dev_id": "pci_0000_00_18_4", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:18.4"}, {"dev_id": "pci_0000_00_18_5", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:18.5"}, {"dev_id": "pci_0000_00_18_6", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:18.6"}, {"dev_id": "pci_0000_00_18_7", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:18.7"}] _report_hypervisor_resource_view /usr/lib/python2.7/site-packages/nova/compute/resource_tracker.py:672
2019-03-29 11:01:45.504 27659 DEBUG oslo_concurrency.lockutils [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Lock "compute_resources" acquired by "nova.compute.resource_tracker._update_available_resource" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2019-03-29 11:01:45.506 27659 DEBUG oslo_messaging._drivers.amqpdriver [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: b7c60a903a7a4a0e8240a37c6941a8d3 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:01:45.640 27659 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: b7c60a903a7a4a0e8240a37c6941a8d3 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:01:45.642 27659 DEBUG nova.scheduler.client.report [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Refreshing aggregate associations for resource provider d3edbe3c-4889-4f14-ab35-ef1bb940aec8 _ensure_resource_provider /usr/lib/python2.7/site-packages/nova/scheduler/client/report.py:402
2019-03-29 11:01:45.678 27659 DEBUG oslo_messaging._drivers.amqpdriver [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 59f9b8422a58402da4de2b70393a2c9b exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:01:45.693 27659 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 59f9b8422a58402da4de2b70393a2c9b __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:01:45.708 27659 DEBUG nova.scheduler.client.report [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Deleting stale allocation for instance 4ad84d2b-babc-4ef6-87eb-437ba51a77dc remove_deleted_instances /usr/lib/python2.7/site-packages/nova/scheduler/client/report.py:750
2019-03-29 11:01:45.742 27659 INFO nova.scheduler.client.report [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Deleted allocation for instance 4ad84d2b-babc-4ef6-87eb-437ba51a77dc
2019-03-29 11:01:45.743 27659 DEBUG oslo_messaging._drivers.amqpdriver [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: f8ac4a03cea34d0ba7aec5ebde1bbdca exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:01:45.775 27659 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: f8ac4a03cea34d0ba7aec5ebde1bbdca __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:01:45.776 27659 DEBUG nova.compute.resource_tracker [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Total usable vcpus: 2, total allocated vcpus: 0 _report_final_resource_view /usr/lib/python2.7/site-packages/nova/compute/resource_tracker.py:688
2019-03-29 11:01:45.777 27659 INFO nova.compute.resource_tracker [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Final resource view: name=com58 phys_ram=8191MB used_ram=512MB phys_disk=299GB used_disk=0GB total_vcpus=2 used_vcpus=0 pci_stats=[]
2019-03-29 11:01:45.784 27659 DEBUG oslo_messaging._drivers.amqpdriver [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 52f70db074444597b3398d53626c7a20 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:01:45.818 27659 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 52f70db074444597b3398d53626c7a20 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:01:45.820 27659 DEBUG nova.scheduler.client.report [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Refreshing aggregate associations for resource provider d3edbe3c-4889-4f14-ab35-ef1bb940aec8 _ensure_resource_provider /usr/lib/python2.7/site-packages/nova/scheduler/client/report.py:402
2019-03-29 11:01:45.847 27659 DEBUG nova.compute.resource_tracker [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Compute_service record updated for com58:com58 _update_available_resource /usr/lib/python2.7/site-packages/nova/compute/resource_tracker.py:626
2019-03-29 11:01:45.848 27659 DEBUG oslo_concurrency.lockutils [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Lock "compute_resources" released by "nova.compute.resource_tracker._update_available_resource" :: held 0.344s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2019-03-29 11:01:45.853 27659 DEBUG oslo_messaging._drivers.amqpdriver [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CAST unique_id: 6efba0961bde45b0be2a99c408c1acc2 FANOUT topic 'scheduler' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:539
2019-03-29 11:01:45.855 27659 WARNING nova.image.glance [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] No protocol specified in for api_server 'http://172.30.126.56:9292', please update [glance] api_servers with fully qualified url including scheme (http / https)
2019-03-29 11:01:45.856 27659 INFO nova.compute.manager [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Migrating instance to com57 finished successfully.
2019-03-29 11:01:45.857 27659 DEBUG oslo_messaging._drivers.amqpdriver [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CAST unique_id: 55e23dbe5b2b41f2b56ec2bcee7cfb15 exchange 'nova' topic 'consoleauth' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:01:45.860 27659 DEBUG oslo_messaging._drivers.amqpdriver [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 633bfe17c44340bf9b0f2ffdde028f0a exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:01:45.902 27659 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 633bfe17c44340bf9b0f2ffdde028f0a __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:01:45.903 27659 DEBUG nova.virt.libvirt.driver [req-001695db-a91b-4779-b890-7c4091a4ba3a 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Live migration monitoring is all done _live_migration /usr/lib/python2.7/site-packages/nova/virt/libvirt/driver.py:6620
2019-03-29 11:01:53.882 27659 DEBUG oslo_messaging._drivers.amqpdriver [-] CALL msg_id: f2ae4b1d1a3441a69ef6bf2f8a40dbee exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:01:53.929 27659 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: f2ae4b1d1a3441a69ef6bf2f8a40dbee __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:01:55.588 27659 DEBUG nova.virt.driver [-] Emitting event <LifecycleEvent: 1553828500.59, 4ad84d2b-babc-4ef6-87eb-437ba51a77dc => Stopped> emit_event /usr/lib/python2.7/site-packages/nova/virt/driver.py:1438
2019-03-29 11:01:55.589 27659 INFO nova.compute.manager [-] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] VM Stopped (Lifecycle Event)
2019-03-29 11:01:55.590 27659 DEBUG oslo_messaging._drivers.amqpdriver [req-e8a2e84a-373d-4286-9ccb-f013bb95a8c0 - - - - -] CALL msg_id: 6f388261ce6a492f80c9535a790a2a5f exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:01:55.635 27659 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 6f388261ce6a492f80c9535a790a2a5f __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:01:55.636 27659 DEBUG nova.compute.manager [req-e8a2e84a-373d-4286-9ccb-f013bb95a8c0 - - - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Checking state _get_power_state /usr/lib/python2.7/site-packages/nova/compute/manager.py:1176

